test/cpp/api/tensor_options.cpp

```diff
@@ -66,18 +66,18 @@ TEST(TensorOptionsTest, ConstructsWellFromCPUTypes) {
   options = TensorOptions(kInt);
   REQUIRE_OPTIONS(kCPU, -1, kInt, kStrided);
 
-  options = TensorOptions(getNonVariableType(Backend::SparseCPU, kFloat));
+  options = TensorOptions(kCPU).dtype(kFloat).layout(kSparse);
```

Comment:
nit: how come sometimes you do TensorOptions(kCPU) and other times device(kCPU) to start?